internal/tftime/resource_time_sleep_test.go

```diff
@@ -75,12 +80,6 @@ func TestAccTimeSleep_CreateDuration(t *testing.T) {
 					testExtractResourceAttr(resourceName, "id", &time1),
 				),
 			},
-			// {
-			// 	ResourceName:      resourceName,
-			// 	ImportState:       true,
-			// 	ImportStateIdFunc: testAccTimeSleepImportStateIdFunc(resourceName),
-			// 	ImportStateVerify: true,
-			// },
```

Comment:
About all those commented out tests that exercised the Import logic: I remember seeing a chat/slack-thread between you and @bflad on this.

I assume the conclusion was to remove it.

Any chance for a later comeback? I haven't checked if we have indeed other tests that still exercise the import, but it makes me nervous to have a functionality that we don't test.

Maybe a bug that needs fixing?

Reply:
Looks like the import tests are only commented in `resource_time_sleep_test.go` because of the use of `d.SetId(time.Now().UTC().Format(time.RFC3339))` in the importer.

Are these tests going to be removed?